datamodels/2.x/itop-core-update/dictionaries/pt_br.dict.itop-core-update.php

```diff
@@ -21,98 +21,98 @@
+    'iTopUpdate:UI:WithoutBackup' => 'Backup não planejado',
+    'iTopUpdate:UI:Backup' => 'Backup gerado antes da atualização',
+	'iTopUpdate:UI:DoFilesArchive' => 'Arquivar arquivos da aplicação',
+	'iTopUpdate:UI:UploadArchive' => 'Escolha um pacote para enviar',
+	'iTopUpdate:UI:ServerFile' => 'Caminho para o pacote já no servidor',
+	'iTopUpdate:UI:WarningReadOnlyDuringUpdate' => 'Durante a atualização, a aplicação ficará em modo leitura.',
 
-    'iTopUpdate:UI:Status' => 'Status~~',
-    'iTopUpdate:UI:Action' => 'Update~~',
-    'iTopUpdate:UI:History' => 'Versions History~~',
-    'iTopUpdate:UI:Progress' => 'Progress of the upgrade~~',
+    'iTopUpdate:UI:Status' => 'Status',
+    'iTopUpdate:UI:Action' => 'Atualizar',
+    'iTopUpdate:UI:History' => 'Versões anteriores',
+    'iTopUpdate:UI:Progress' => 'Progresso da atualização',
 
-    'iTopUpdate:UI:DoBackup:Label' => 'Backup files and database~~',
-    'iTopUpdate:UI:DoBackup:Warning' => 'Backup is not recommended due to limited available disk space~~',
+    'iTopUpdate:UI:DoBackup:Label' => 'Backup de arquivos e base de dados',
+    'iTopUpdate:UI:DoBackup:Warning' => 'Backup não recomendado devido ao espaço em disco limitado',
 
-    'iTopUpdate:UI:DiskFreeSpace' => 'Disk free space~~',
-    'iTopUpdate:UI:ItopDiskSpace' => 'iTop disk space~~',
-    'iTopUpdate:UI:DBDiskSpace' => 'Database disk space~~',
-	'iTopUpdate:UI:FileUploadMaxSize' => 'File upload max size~~',
+    'iTopUpdate:UI:DiskFreeSpace' => 'Espaço em disco disponível',
+    'iTopUpdate:UI:ItopDiskSpace' => 'Espaço em disco do iTop',
+    'iTopUpdate:UI:DBDiskSpace' => 'Espaço em disco da base de dados',
+	'iTopUpdate:UI:FileUploadMaxSize' => 'Tamanho máximo de envio de arquivos',
 
-	'iTopUpdate:UI:PostMaxSize' => 'PHP ini value post_max_size: %1$s~~',
-	'iTopUpdate:UI:UploadMaxFileSize' => 'PHP ini value upload_max_filesize: %1$s~~',
+	'iTopUpdate:UI:PostMaxSize' => 'PHP ini post_max_size: %1$s',
+	'iTopUpdate:UI:UploadMaxFileSize' => 'PHP ini upload_max_filesize: %1$s',
 
-    'iTopUpdate:UI:CanCoreUpdate:Loading' => 'Checking filesystem~~',
-    'iTopUpdate:UI:CanCoreUpdate:Error' => 'Checking filesystem failed (%1$s)~~',
-    'iTopUpdate:UI:CanCoreUpdate:ErrorFileNotExist' => 'Checking filesystem failed (File not exist %1$s)~~',
-    'iTopUpdate:UI:CanCoreUpdate:Failed' => 'Checking filesystem failed~~',
-    'iTopUpdate:UI:CanCoreUpdate:Yes' => 'Application can be updated~~',
-	'iTopUpdate:UI:CanCoreUpdate:No' => 'Application cannot be updated: %1$s~~',
-	'iTopUpdate:UI:CanCoreUpdate:Warning' => 'Warning: application update can fail: %1$s~~',
+    'iTopUpdate:UI:CanCoreUpdate:Loading' => 'Verificando arquivos de sistema',
+    'iTopUpdate:UI:CanCoreUpdate:Error' => 'Falha ao verificar arquivos de sistema (%1$s)',
+    'iTopUpdate:UI:CanCoreUpdate:ErrorFileNotExist' => 'Falha ao verificar arquivos de sistema (arquivo não existe %1$s)~~',
```

Comment:
```suggestion
    'iTopUpdate:UI:CanCoreUpdate:ErrorFileNotExist' => 'Falha ao verificar arquivos de sistema (arquivo não existe %1$s)',
```

'~~' needs to be removed

Reply:
Fixed